Mathlib/MeasureTheory/Function/SimpleFuncDenseLp.lean

```diff
@@ -229,14 +229,17 @@ theorem integrable_approxOn [BorelSpace E] {f : β → E} {μ : Measure β} (fme
   rw [← memℒp_one_iff_integrable] at hf hi₀ ⊢
   exact memℒp_approxOn fmeas hf h₀ hi₀ n
 
-theorem tendsto_approxOn_range_L1_nnnorm [OpensMeasurableSpace E] {f : β → E} {μ : Measure β}
+theorem tendsto_approxOn_range_L1_enorm [OpensMeasurableSpace E] {f : β → E} {μ : Measure β}
```

Comment:
I like the renaming, but please also fix the lemma to match up. (I only reviewed up to 6e98128ca8a15da4fcb5e574f11f558115350805; perhaps you're changing this later already.)

Reply:
Okay, sounds like 4d891a448293449c457d3a1a28832e0fc90d295f fixed this, thanks :-)